### ISSUE TITLE
support all VDV-453/-454 services

### ISSUE THREAD

**Issue by derhuerst** [OpenDataVBB/vdv-453-client#4]
This client should support *all* VDV-453/VDV-454 services, not just those that VBB currently uses.

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/index.js#L19-L30

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/index.js#L51-L56

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/index.js#L70-L76

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/index.js#L543-L550

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/index.js#L738-L751

https://github.com/OpenDataVBB/vdv-453-client/blob/0ace9f3a48b981278f78107fd5fbeead4b09ccfa/lib/server.js#L17-L37

**Comment by dancesWithCycles** [comment]
How about an on-demand procedure? I reckon `AUS` is the most important service, isn't it? If my memory services me right, the `AUS` service delivers the realtime information.

**Comment by derhuerst** [comment]
> How about an on-demand procedure?

> I reckon `AUS` is the most important service, isn't it? If my memory services me right, the `AUS` service delivers the realtime information.

I agree: `DFI` (for realtime arrivals/departures`) and `ANS` (trip-based realtime data) seem relevant to me as soon as `AUS` is not supported.
